### DISCUSSION TITLE
小白提问，关于全参预训练

### DISCUSSION THREAD

**Discussion by haonit** [ymcui/Chinese-LLaMA-Alpaca-3#81] [Q&A]
请问wiki的faq，问题5：为什么不对模型做全量预训练而是用LoRA？  但是在readme里“[下载地址](https://github.com/ymcui/Chinese-LLaMA-Alpaca-3?tab=readme-ov-file#%E4%B8%8B%E8%BD%BD%E5%9C%B0%E5%9D%80)”板块，有Llama-3-Chinese-8B完整版呀。按照我的理解，这个完整版是从Meta的llama-3-8B base模型做全参中文无监督预训练得到的。
小白理解有限，谁能解释一下么？十分感谢

**Comment by ymcui** [comment]
完整版指的是已经将LoRA和base模型合并，直接可以使用的模型，并不是说模型是`全量精调`得来的。
